program/src/state/config.rs

```diff
@@ -0,0 +1,52 @@
+use bytemuck::{Pod, Zeroable};
+use shank::ShankAccount;
+use solana_program::pubkey::Pubkey;
+use spl_discriminator::SplDiscriminate;
+use spl_pod::optional_keys::OptionalNonZeroPubkey;
```

Comment:
micro-nit: I'm not sure if I've missed this before, but we usually group these together. You can set up `rustfmt` using:

```
group_imports = "One"
imports_granularity = "One"
```

Reply:
Yeah, I need to set up the `nightly` formatter.